crossfit/backend/torch/hf/memory_curve_utils.py

```diff
@@ -29,7 +30,7 @@
 
 
 def fit_memory_estimate_curve(
-    model: PreTrainedModel,
+    model: "transformers.PreTrainedModel",
```

Comment:
The `""` quotes might not be needed if you're using `from __future__ import annotations`

Reply:
I am not sure, my understanding is we need this in python<3.10. I am trying to be cautious here following this standard. 

https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations

Reply:
Keeping as string for now